Specs/DateTime/Chinese/DateParser.json

```diff
@@ -724,14 +724,15 @@
     "Context": {
       "ReferenceDateTime": "2017-03-22T00:00:00"
     },
+    "NotSupported": "java, javascript, python",
     "Results": [
       {
         "Text": "本月十日",
         "Type": "date",
         "Value": {
-          "Timex": "XXXX-03-10",
```

Comment:
Technicall the most correct timex here would be XXXX-XX-10, right? Is the behavious in English the same?

Reply:
added comment